builtin/providers/google/resource_compute_vpn_tunnel_test.go

```diff
@@ -32,6 +32,22 @@ func TestAccComputeVpnTunnel_basic(t *testing.T) {
 	})
 }
 
+func TestAccComputeVpnTunnel_noopdiff(t *testing.T) {
```

Comment:
Since pretty much every test automatically is checking for a no-op diff, how about we instead call this `TestAccComputeVpnTunnel_defaultTrafficSelectors`?